GWAS/GWAS_QC.ipynb

```diff
@@ -483,11 +485,11 @@
     "bash: container=container_lmm, volumes=[f'{cwd}:{cwd}'], expand= \"${ }\", stderr = f'{_output:n}.stderr', stdout = f'{_output:n}.stdout'\n",
     "    plink2 \\\n",
     "      --bfile ${_input:n} \\\n",
-    "      ${('--maf %s' % maf_filter) if maf_filter >= 0 else ''} \\\n",
+    "      ${('--maf %s' % maf_filter) if maf_filter > 0 else ''} \\\n",
```

Comment:
@dianacornejo are you sure to remove `=` ?  how do we want to include all variants without filtering -- if we specify `maf _filter = 0` it is going to just ignore it with the `else` statement, then will PLINK use some default, non-zero values instead? Or it's going to use `--maf 0` by default?

Reply:
Sorry I didn't mean to change this